vms/platformvm/txs/executor/proposal_tx_executor.go

```diff
@@ -246,8 +235,6 @@ func (e *ProposalTxExecutor) AddDelegatorTx(tx *txs.AddDelegatorTx) error {
 	avax.Consume(e.OnAbortState, tx.Ins)
 	// Produce the UTXOs
 	avax.Produce(e.OnAbortState, txID, onAbortOuts)
-
-	e.PrefersCommit = tx.StartTime().After(e.Clk.Time())
```

Comment:
I'm assuming we can remove these because they are no longer included in proposal blocks?

Reply:
Yeah - only `RewardValidatorTx`s exist in proposal blocks now.